bridges/CeskaTelevizeBridge.php

```diff
@@ -38,7 +38,7 @@ private function getUploadTimeFromString($string) {
 	public function collectData() {
 		$url = $this->getInput('url');
 
-		$validUrl = '/^(https:\/\/www\.ceskatelevize\.cz\/porady\/\d+-[a-z0-9-]+\/)(dily\/((nove|vysilani)\/)?)?$/';
+		$validUrl = '/^(https:\/\/www\.ceskatelevize\.cz\/porady\/\d+-[a-z0-9-]+\/)(bonus\/)?$/';
```

Comment:
Since regex of validUrl is changed, could you change exampleValue on line 16?

Reply:
True, I forgot. Fixed in following commit.